### ISSUE TITLE
Advice for other nozzle sizes

### ISSUE THREAD

**Issue by wyldbill** [eoprede/prusa_profiles#16]
Do the maintainers have any guidance for using these profiles with non .4 mm nozzles? .6 and .25 are common alternatives. Could there be an updated or alternate README?

**Comment by trevjonez** [comment]
The profiles are setup for .4mm nozzle size. if you look at the plain text in the ini files in github for prusaslicer and search for  `extrusion_width` you will see the profiles spec out the exact width of those setting for the config, which gives you a good idea of the nozzle size it was setup for usually.

Additionally, things like temperature and speed are going to be different for other nozzle sizes, so they are very much specific to the nozzle in the printer config. Different because there is likely a deterministic formula to estimate best temperature for volume of material per mm^2 extruded. where speed, nozzle size, extruder design/behavior, and material properties are all variables. Seemingly an infinite combination of values feeding into the model.

This advice is my own based on my own experience which stems from my experience with prusaslicer. Perhaps other slicers have modeled their profiles in a more abstract way so the coupling between printer configuration and material configuration is not an issue, though I doubt it.

There are just far too many variables to make any general formulation to what is best to be worthwhile. Which I think is because what is best for my stl, printer, and opinion, will not be a match for your own opinion, printer, or stl.


TLDR: The profiles are for .4 nozzles on printers matching the authors own printer. You will need to tune your own profile for different nozzle sizes.
